client/src/components/PacmanGame/gameCore.js

```diff
@@ -23,7 +17,7 @@ const wallGenerator = numberofCells => {
     wall.push(topRow);
     wall.push(bottomRow);
   }
-  for (let row = 0; row < numberofCells; row++) {
+  for (let row = 0; row < numberofCells; row += 1) {
```

Comment:
separate function for this.

Reply:
Made separate function but this will later be removed to functions to generate the real Pacman wall layout rather than the current arbitrary pattern.